test/libsolidity/Imports.cpp

```diff
@@ -104,8 +104,7 @@ BOOST_AUTO_TEST_CASE(simple_alias)
 BOOST_AUTO_TEST_CASE(library_name_clash)
 {
 	CompilerStack c;
-	c.addSource("a", "library A {} pragma solidity >=0.0;");
-	c.addSource("b", "library A {} pragma solidity >=0.0;");
+	c.addSource("a", "library A {} library A {} pragma solidity >=0.0;");
```

Comment:
At this point, the libraries are actually internally called "a:A" and "b:A" and would be referenced as `import A from "a"` and `import A from "b"`.  The only way I could think to cause clashes would be to introduce a third source with imports the two of them (both `as A`), but I am presuming, perhaps incorrectly, that the compiler would notice this and give an error before getting to a linker.  So, I'm not completely sure how to set up this test correctly.